devtools/cli/src/main/java/io/quarkus/cli/build/GradleRunner.java

```diff
@@ -235,9 +235,12 @@ void setGradleProperties(ArrayDeque<String> args, boolean batchMode) {
     }
 
     void verifyBuildFile() {
-        File buildFile = projectRoot.resolve("build.gradle").toFile();
-        if (!buildFile.isFile()) {
-            throw new IllegalStateException("Was not able to find a build file in: " + projectRoot);
+        for (String buildFileName : buildTool.getBuildFiles()) {
+            File buildFile = projectRoot.resolve(buildFileName).toFile();
+            if (buildFile.exists()) {
+                return;
+            }
         }
+        throw new IllegalStateException("Was not able to find a build file in: " + projectRoot);
```

Comment:
Maybe we could show the list of required build files here?

Reply:
Yes this is a good idea, I pushed an update.